src/openvic-simulation/country/CountryInstance.cpp

```diff
@@ -1485,6 +1486,10 @@ void CountryInstance::country_reset_before_tick() {
 	for (auto pair : goods_data) {
 		pair.second.clear_daily_recorded_data();
 	}
+
+	for (auto pair : taxable_income_by_pop_type) {
+		pair.second = fixed_point_t::_0();
+	}
```

Comment:
```suggestion
	taxable_income_by_pop_type.clear();
```
You could also do `taxable_income_by_pop_type.fill(fixed_point_t::_0());`, `clear` default-initialises every value while `fill` copies the arg into every value